theano/sandbox/gpuarray/opt.py

```diff
@@ -260,6 +261,25 @@ def local_gpu_elemwise(node):
     return res
 
 
+@register_opt()
+@gof.local_optimizer([GpuElemwise])
+def cast_elemwise_pow_exponent(node):
```

Comment:
I just updated the description. I just wanted to create the PR as soon as possible so Travis would start faster. Let me know if anything is still needed